package.json

```diff
@@ -24,6 +24,14 @@
       "types": "./dist/runtime/index.d.ts",
       "import": "./dist/runtime/index.mjs"
     },
+    "./runtime/components/MDCRenderer": {
```

Comment:
Let's add `components` directory in `exports` and not just `MDCRenderer`.

Reply:
I don't believe this is possible with the current state of the repo and exported types. 

There is no generated `runtime/components/index.mjs` so we can't import directly from there. Also, the `MDC.vue` component and none of the `runtime/components/prose/*.vue` files are generating types in the build.

If we created a `runtime/components/index.ts` file that exported all of the components (a barrel file) it would then trigger the proper exports, but could have an impact on tree-shaking; however, the other components like `MDC.vue` utilize Nuxt-specific imports, meaning it would actually break using in a non-Nuxt project.

Since only the `MDCRenderer.vue` component was "cleaned" of Nuxt imports, exporting anything else as-is would break 😬 

Good with this for now, or any ideas?

Reply:
Why not just use `"./runtime/components/*": "./dist/runtime/components/*",` in package exports? Then we can import like `import MDCRenderer from '@nuxtjs/mdc/runtime/components/MDCRenderer.vue';`

Vitest and Vue test-utils seems fine for me 👍

Reply:
@farnabaz I adjusted the imports in d3108ec3228ad753c8aedb8dd62b2120b2e847bf to this:

```json
"./runtime/components/*": {
  "types": "./dist/runtime/components/*.d.ts",
  "import": "./dist/runtime/components/*"
},
"./dist/runtime/components/*": {
  "types": "./dist/runtime/components/*.d.ts",
  "import": "./dist/runtime/components/*"
}
```

For the `types` property, I had to add `*.d.ts` in order to properly resolve the component types. 

![declaration file error](https://github.com/nuxt-modules/mdc/assets/2229946/43f930e0-e29d-410c-b738-d3c61822b16d)

Technically this doesn't allow for importing types from the nested `/runtime/components/prose/*` components, but the `build` command doesn't generate declaration files for these components anyway, so I think it's fine?

Reply:
LGTM 👍 
We can update those components later to generate ts file